steps/previsit/attempt-visit.ts

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+import {
+  ComponentWrapper,
+  DatabaseMap,
+  DynamicComponent,
+  StaticComponent
+} from "remultiform/component-wrapper";
+import {
+  Heading,
+  HeadingLevels
+} from "lbh-frontend-react/components/typography/Heading";
+
+import { Paragraph } from "lbh-frontend-react/components/typography/Paragraph";
+
+import { ImageInput } from "../../components/ImageInput";
+import { makeSubmit } from "../../components/makeSubmit";
+
+import DatabaseSchema from "../../store/DatabaseSchema";
+import processId from "../../store/processId";
+
+import ProcessStepDefinition from "../../components/ProcessStepDefinition";
+
+const step: ProcessStepDefinition = {
+  title: "Attempt visit",
+  heading: "Tenancy and Household Check",
+  step: {
+    slug: "attempt-visit",
+    nextSlug: "submit",
+    Submit: makeSubmit({ href: "/submit", value: "Enter the property" }),
```

Comment:
Let's also add the link to the unable to enter journey (even if it doesn't go anywhere).